test/mitmproxy/contentviews/test_image_parser.py

```diff
@@ -20,7 +18,9 @@
         ('Title', 'PngSuite'),
         ('Author', 'Willem A.J. van Schaik\n(willem@schaik.com)'),
         ('Copyright', 'Copyright Willem van Schaik, Singapore 1995-96'),
-        ('Description', 'A compilation of a set of images created to test the\nvarious color-types of the PNG format. Included are\nblack&white, color, paletted, with alpha channel, with\ntransparency formats. All bit-depths allowed according\nto the spec are present.'), ('Software', 'Created on a NeXTstation color using "pnmtopng".'), ('Disclaimer', 'Freeware.')
+        ('Description', 'A compilation of a set of images created to test the\nvarious color-types of the PNG format. Included are\nblack&white, color, paletted, with alpha channel, with\ntransparency formats. All bit-depths allowed according\nto the spec are present.'),
```

Comment:
The only lint errors are that these lines are > 140 characters. Doing it like this
```
    A compilation of a \
    set of images ...
```
causes extra spaces in the sentence, and removing the spaces will kill the indentation. So I left it as it is.